packages/contracts-bedrock/src/Safe/LivenessModule.sol

```diff
@@ -132,11 +135,16 @@ contract LivenessModule is ISemver {
     /// @param _ownersToRemove The owners to remove
     function removeOwners(address[] memory _previousOwners, address[] memory _ownersToRemove) external {
         require(_previousOwners.length == _ownersToRemove.length, "LivenessModule: arrays must be the same length");
+        address[] memory currentOwners = SAFE.getOwners();
+        require(
+            !deactivated,
+            "LivenessModule: The safe has been shutdown, the LivenessModule should be removed or replaced."
```

Comment:
What do you think about this suggestion? Technically it should be fine to also keep the guard, but in the case of a shutdown, we want to at least consider removing/replacing it, hence this suggestion
```suggestion
            "LivenessModule: The safe has been shutdown, the LivenessModule and LivenessGuard should be removed or replaced."
```